modules/govuk_ci/manifests/master.pp

```diff
@@ -57,4 +59,9 @@
     mode => '0555',
   }
 
+  create_resources('::govuk_jenkins::ssh_slave', $ci_agents, {
+    'credentials_id' => $credentials_id,
+    'labels'         => 'mongo-3.2',
```

Comment:
Is this to ensure that all CI hosts have this version of mongo? Should this be in the hiera with the other labels?